src/obs_interface.cpp

```diff
@@ -224,10 +224,11 @@ void ObsInterface::create_output(const std::string& recordingPath, bool bufferin
   } else {
     blog(LOG_INFO, "Set ffmpeg_muxer settings");
 		obs_data_set_string(settings, "extension", "mp4");
-    // Apparently need to specify the exact path for ffmpeg_muxer.
-    // TODO add something to auto generate this ?
-    obs_data_set_string(settings, "path", (recordingPath + "/noobs.mp4").c_str()); 
-    recording_path = recordingPath + "/noobs.mp4";
+    // Need to specify the exact path for ffmpeg_muxer.
+    std::string filename = recordingPath + "/" + get_current_date_time() + ".mp4";
+    obs_data_set_string(settings, "path", filename.c_str());
+    recording_path = filename;
```

Comment:
It was like this before but should we be assigning the value here? It seems like only one branch of the if() is writing to this

Reply:
not sure how else we would do this - although probably worth a read of the libobs source here to see why the method for replay_buffer is much better (for both setting the path and getting the last recording) - there might be a better way.